api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2084,6 +2071,52 @@ type ContainerOverrides struct {
 	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
 }
 
+// ImageConfig provides a policy for customizing an image.
+type ImageConfig struct {
```

Comment:
We should add some documentation here that the values are merged if multiple entries are matching.